fr/posts/how-money-moved/index.html

```diff
@@ -0,0 +1,191 @@
+  const metamaskTypeForm = 'https://eco-network.typeform.com/to/Igx6jFCz';
+  const defaultTypeForm = 'https://eco-network.typeform.com/to/z6zCdWjw';
+  const typeFormToRedirect = (typeof web3 !== 'undefined') ? metamaskTypeForm : defaultTypeForm 
+  
+function metamaskForm() {
+  if (typeof web3 !== 'undefined') {
+    var forms = document.getElementsByClassName('typeform-form');
+    for (var i = 0; i < forms.length; ++i) {
+      var form = forms[i];
+      form.setAttribute('redirect', typeFormToRedirect);
+      form.setAttribute('action', typeFormToRedirect);
+    }
+  }
+} 
+
+if( document.readyState !== 'loading' ) {
+    metamaskForm();
+} else {
+    document.addEventListener('DOMContentLoaded', function () {
+        metamaskForm();
+    });
+} 
+</script>
+
+<script>
+  var Webflow = Webflow || [];
+
+  Webflow.push(function() {  
+    $(document).off('submit');
+
+    const emailField = $('.form_text-field');
+
+    async function postData(url = '', data = {}) {
+      const response = await fetch(url, {
+        method: 'POST',
+        mode: 'no-cors', 
+        cache: 'no-cache',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        referrerPolicy: 'strict-origin-when-cross-origin',
+        body: JSON.stringify(data)
+      });
+      return response;
+    }
+
+    $('form').submit(function(evt) {
+      evt.preventDefault();
+      const enteredEmail = emailField.val()
+
+      //This is production endpoint.
+      const waitlistUrl = 'https://836gwkd0c3.execute-api.us-west-1.amazonaws.com/production/waitlist';
+     
+      postData(waitlistUrl, { email: enteredEmail })
+        .then(data => {
+          window.location = typeFormToRedirect + '?email=' + enteredEmail;
+        })
+        .catch((error) => {
+          console.error('Error:', error);
+        });
+      
+    });
+  });
+</script>
+</body></html>
```

Comment:
You should add the header disclaimer injections script here : 
https://github.com/eco-community/website-translations#how-to-add-the-non-official-warning-disclaimer